util/reflect.go

```diff
@@ -621,10 +621,12 @@ type FieldIteratorFunc func(ni *NodeInfo, in, out interface{}) Errors
 // leafref. Fields that are present in value that are explicitly noted not to
 // have a corresponding schema (e.g., annotation/metadata fields added by ygen)
 // are skipped during traversal.
-//   schema is the schema corresponding to value.
-//   in, out are passed to the iterator function and can be used to carry state
-//     and return results from the iterator.
-//   iterFunction is executed on each scalar field.
+//
+//	schema is the schema corresponding to value.
+//	in, out are passed to the iterator function and can be used to carry state
+//	  and return results from the iterator.
+//	iterFunction is executed on each scalar field.
+//
```

Comment:
Change to a [list](https://go.dev/doc/comment#lists) instead of the current erroneous indenting.